cli/index.ts

```diff
@@ -347,23 +388,25 @@ export function runCli() {
           },
           schemaSuffixOverrideOption,
           jsonOutputOption,
-          varsOption
+          varsOption,
+          timeoutOption
```

Comment:
You might want to add this to the run command too?

Reply:
whoops, thanks